s3/pkg/datastore/gcp/gcp.go

```diff
@@ -53,8 +53,7 @@ func (ad *GcsAdapter) BucketDelete(ctx context.Context, input *pb.Bucket) error
 
 func (ad *GcsAdapter) BucketCreate(ctx context.Context, input *pb.Bucket) error {
 	bucket := ad.session.NewBucket()
-	acl := models.ACL(input.Acl.CannedAcl)
-	err := bucket.Create(input.Name, acl)
+	err := bucket.Create(input.Name, models.PublicReadWrite)
```

Comment:
this change doesn't seem to be relevant to the PR

Reply:
Why Public RW?

Reply:
since private buckets r not allowing to upload objects